src/test/java/hudson/plugins/ansicolor/action/ColorizedActionTest.java

```diff
@@ -60,9 +60,18 @@ public void willReturnDefaultIfLogDoesntContainAnnotation() {
     }
 
     @Test
-    public void willReturnDefaultIfLogAnnotationPointsToNonexistingAction() {
+    public void willReturnDefaultIfLogAnnotationPointsToNonexistentAction() {
         final MarkupText markupText = new MarkupText("Log line");
         markupText.addMarkup(0, TAG_ACTION_BEGIN + "\"identifier_not_in_actions\"" + TAG_ACTION_END);
         assertEquals(CONTINUE, ColorizedAction.parseAction(markupText, buildRun));
     }
+
+    @Test
+    public void willReturnCommandIgnoreOnPipelineInternalLine() {
+        final MarkupText markupText = new MarkupText("Some internal line");
+        markupText.addMarkup(0, "<span class=\"pipeline-new-node\">");
+        System.out.println(markupText.toString(false));
```

Comment:
Debugging? Remove.

Reply:
Oh the shame ;-)
Believe it or not I **never** do this.
Removed.